inc/url_replacer.php

```diff
@@ -169,16 +172,22 @@ public function build_image_url(
 		}
 		$args = apply_filters( 'optml_image_args', $args, $original_url );
 
-		$new_url = ( new Optml_Image( $url, $args ) )->get_url(
-			[
-				'signed'          => $this->settings->use_lazyload() ? false : $this->is_allowed_site,
-				'apply_watermark' => apply_filters( 'optml_apply_watermark_for', true, $url ),
-			]
-		);
+		$arguments = [
+			'signed'          => $this->settings->use_lazyload() ? false : $this->is_allowed_site,
+			'apply_watermark' => apply_filters( 'optml_apply_watermark_for', true, $url ),
+
+		];
+
+		if ( isset( $args['format'] ) && ! empty( $args['format'] ) ) {
```

Comment:
it makes more sense to me to have the format in `$args` rather `$arguments`. I have imagined `$arguments` those more like global settings applied to all the images, even if the variable name is not that intuitive 🙉